Unable to start EmbeddedMysql on Window 7 using v5_7_latest

Using that latest release of wix-embedded-mysql 3.0.0 I get the following exception running my unit test when using v5_7_latest for mysql version.  Using v5_6_latest works fine.  I have not tested any other versions.

java.lang.RuntimeException: java.io.IOException: java.lang.RuntimeException: mysql start failed with error: [ERROR] Cannot open Windows EventLog; check privileges, or start server with --log_syslog=0
2018-01-18T22:59:39.420261Z 0 [Note] C:\git\github.snei.sony.

	at com.wix.mysql.EmbeddedMysql.<init>(EmbeddedMysql.java:51)
	at com.wix.mysql.EmbeddedMysql$Builder.start(EmbeddedMysql.java:155)
	at com.sony.sie.cs.test.unit.datasource.MysqlEmbeddedDatabaseBuilder.createDatabase(MysqlEmbeddedDatabaseBuilder.java:60)
	at com.sony.sie.cs.test.unit.datasource.MysqlEmbeddedDatabaseBuilder.build(MysqlEmbeddedDatabaseBuilder.java:88)
	at com.sony.sie.cs.test.unit.datasource.MySqlEmbeddedDataSourceWrapper.reset(MySqlEmbeddedDataSourceWrapper.java:50)
	at com.sony.sie.cs.csapi.service.ingestion.IngestionServiceImplTest.setup(IngestionServiceImplTest.java:127)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.internal.runners.MethodRoadie.runBefores(MethodRoadie.java:133)
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:96)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.executeTest(PowerMockJUnit44RunnerDelegateImpl.java:294)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.executeTestInSuper(PowerMockJUnit47RunnerDelegateImpl.java:127)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.executeTest(PowerMockJUnit47RunnerDelegateImpl.java:82)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.runBeforesThenTestThenAfters(PowerMockJUnit44RunnerDelegateImpl.java:282)
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:87)
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:50)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.invokeTestMethod(PowerMockJUnit44RunnerDelegateImpl.java:207)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.runMethods(PowerMockJUnit44RunnerDelegateImpl.java:146)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$1.run(PowerMockJUnit44RunnerDelegateImpl.java:120)
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:34)
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:44)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.run(PowerMockJUnit44RunnerDelegateImpl.java:122)
	at org.powermock.modules.junit4.common.internal.impl.JUnit4TestSuiteChunkerImpl.run(JUnit4TestSuiteChunkerImpl.java:106)
	at org.powermock.modules.junit4.common.internal.impl.AbstractCommonPowerMockRunner.run(AbstractCommonPowerMockRunner.java:53)
	at org.powermock.modules.junit4.PowerMockRunner.run(PowerMockRunner.java:59)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: java.io.IOException: java.lang.RuntimeException: mysql start failed with error: [ERROR] Cannot open Windows EventLog; check privileges, or start server with --log_syslog=0
2018-01-18T22:59:39.420261Z 0 [Note] C:\git\github.snei.sony.
	at com.wix.mysql.MysqldProcess.onAfterProcessStart(MysqldProcess.java:60)
	at de.flapdoodle.embed.process.runtime.AbstractProcess.<init>(AbstractProcess.java:114)
	at com.wix.mysql.MysqldProcess.<init>(MysqldProcess.java:42)
	at com.wix.mysql.MysqldExecutable.start(MysqldExecutable.java:44)
	at com.wix.mysql.MysqldExecutable.start(MysqldExecutable.java:21)
	at de.flapdoodle.embed.process.runtime.Executable.start(Executable.java:101)
	at com.wix.mysql.EmbeddedMysql.<init>(EmbeddedMysql.java:47)
	... 31 more
Caused by: java.lang.RuntimeException: mysql start failed with error: [ERROR] Cannot open Windows EventLog; check privileges, or start server with --log_syslog=0
2018-01-18T22:59:39.420261Z 0 [Note] C:\git\github.snei.sony.
	at com.wix.mysql.MysqldProcess.onAfterProcessStart(MysqldProcess.java:56)
	... 37 more

--- a/wix-embedded-mysql/src/main/java/com/wix/mysql/distribution/service/Mysql57CommandEmitter.java
+++ b/wix-embedded-mysql/src/main/java/com/wix/mysql/distribution/service/Mysql57CommandEmitter.java
@@ -16,6 +16,6 @@ public class Mysql57CommandEmitter implements CommandEmitter {
 
     @Override
     public List<String> emit(MysqldConfig config, IExtractedFileSet exe) throws IOException {
-        return Collections.newArrayList("--show_compatibility_56=ON");
+        return Collections.newArrayList("--show_compatibility_56=ON", "--log_syslog=0");
     }
 }

